### PR TITLE
docs: add Consolas font and prioritize resource loading

### PR DESCRIPTION
**Pull request by amareshsm**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain:
     In docs website, preload fonts and pre-connect to google tag manager.

#### What changes did you make? (Give an overview)
- Added `Consolas` font:  to override the `@`, `{`, `}`, `(`, and `)` glyphs in Space Mono because they look kind of weird. (similar to https://github.com/eslint/eslint.org/pull/318)

- Prioritized resource loading: Preload font and pre-connect to google tag manager.

### CODE REVIEW

**Review by snitin315 — CHANGES_REQUESTED** [review]
This font file is not present at the moment - https://github.com/eslint/eslint/tree/main/docs/src/assets/fonts

File: docs/src/_includes/layouts/base.html [hunk 1]
```diff
@@ -61,6 +61,8 @@
     <link rel="preload" href="{{ '/assets/fonts/SpaceGrotesk-Medium-subset.woff2' | url }}" as="font" type="font/woff2" crossorigin>
     <link rel="preload" href="{{ '/assets/fonts/Inter-Regular-subset.woff2' | url }}" as="font" type="font/woff2" crossorigin>
     <link rel="preload" href="{{ '/assets/fonts/SpaceMono-Regular-subset.woff2' | url }}" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="{{ '/assets/fonts/Consolas.woff' | url }}" as="font" type="font/woff" crossorigin>
```
Comment (nzakas) [inline docs/src/_includes/layouts/base.html:64]: Consolas was never added to this repo. We'd need to duplicate what's in this PR:

https://github.com/eslint/eslint.org/pull/318

Comment (amareshsm) [reply docs/src/_includes/layouts/base.html:64]: done

**Review by nzakas — APPROVED** [review]
LGTM. Thanks!

**Review by harish-sethuraman — APPROVED** [review]
LGTM

**Comment by nzakas** [comment]
@snitin315 can you re-review please?

**Review by snitin315 — COMMENTED** [review]
My old comment is still valid. https://github.com/eslint/eslint/pull/16225#pullrequestreview-1081382583

You need to port [this `Consolas.woff` file](https://github.com/eslint/eslint.org/pull/318/files#diff-b2625fe9d4a051c6a1d634ffdf2c30348c82c73bf8ec1ae767b4431f77b57a42) as well.

**Review by snitin315 — CHANGES_REQUESTED** [review]
From deploy preview.

**Comment by amareshsm** [comment]
> My old comment is still valid. [#16225 (review)](https://github.com/eslint/eslint/pull/16225#pullrequestreview-1081382583)
> 
> You need to port [this `Consolas.woff` file](https://github.com/eslint/eslint.org/pull/318/files#diff-b2625fe9d4a051c6a1d634ffdf2c30348c82c73bf8ec1ae767b4431f77b57a42) as well.

Done

**Review by snitin315 — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**